### PR TITLE
added customise readme

### PR DESCRIPTION
**Pull request by danielmarostica**
Please tell me what you think. We could add the whole tutorial to a separate file, perhaps.

### CODE REVIEW

File: CUSTOMISE_PORTFOLIO.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Customize it to make your own portfolio ✏️
+
+In this project, there are basically 4 things that you need to change to customize this to anyone else's portfolio: **package.json**, **Personal Information**, **Github Information** and **Splash Logo**.
+
+### package.json
+
+Open this file, which is in the main cloned directory, choose any "name" and change "homepage " to `https://<your-github-username>.github.io`. Do not forget the `https://`, otherwise fonts will not load.
+
+### Personal Information
```
Comment (ashutosh1919) [inline CUSTOMISE_PORTFOLIO.md:9]: Yes we should do this. But if we are doing this then we should add detailed description about each component in `portfolio.js`

Comment (danielmarostica) [reply CUSTOMISE_PORTFOLIO.md:9]: Ok in my free time 😬👍

**Comment by danielmarostica** [comment]
Please check and give feedback about what I have begun with.

**Comment by saiteja13427** [comment]
This looks good but I think it has become too old now and has conflicts.

And also I think we can make the doc even better by adding exact key names and their functionalities.

That will make it straightforward. 

I am closing it now due to inactivity.

@danielmarostica can you please raise a fresh PR with improvements.

I will create a new issue regarding this as well.

**Pull request closed without merging**